Provisioning Global L4 NLB faces runtime error

<!-- Thanks for filing an issue! Before submitting, please fill in the following information. -->

**What happened**
: Global L4 NLB 생성시, 런타임 애러 발생

**How to reproduce it (as minimally and precisely as possible)**
: Global L4 NLB 생성

**Anything else we need to know?**
```
goroutine 10059 [running]:
github.com/cloud-barista/cb-tumblebug/src/core/mcis.GetVmIp({0xc0013284d3, 0x4}, {0xc000c7c730, 0xa}, {0xc000dc11b8, 0x4})
	/home/son/go/src/github.com/cloud-barista/cb-tumblebug/src/core/mcis/manageInfo.go:1040 +0x2aa
github.com/cloud-barista/cb-tumblebug/src/core/mcis.RunRemoteCommand({0xc0013284d3, 0x4}, {0xc000c7c730, 0xa}, {0xc000cd390c, 0x5}, {0x0, 0x0}, {0xc001156120, 0x1, ...})
	/home/son/go/src/github.com/cloud-barista/cb-tumblebug/src/core/mcis/remoteCommand.go:182 +0x193
github.com/cloud-barista/cb-tumblebug/src/core/mcis.RunRemoteCommandAsync(0x0?, {0xc0013284d3, 0x4}, {0xc000c7c730, 0xa}, {0xc000cd390c, 0x5}, {0x0, 0x0}, {0xc001156120, ...}, ...)
	/home/son/go/src/github.com/cloud-barista/cb-tumblebug/src/core/mcis/remoteCommand.go:222 +0x13e
created by github.com/cloud-barista/cb-tumblebug/src/core/mcis.RemoteCommandToMcis
	/home/son/go/src/github.com/cloud-barista/cb-tumblebug/src/core/mcis/remoteCommand.go:160 +0x791
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xddffea]

goroutine 10060 [running]:
github.com/cloud-barista/cb-tumblebug/src/core/mcis.GetVmIp({0xc0013284d3, 0x4}, {0xc000c7c730, 0xa}, {0xc000e7b538, 0x4})
	/home/son/go/src/github.com/cloud-barista/cb-tumblebug/src/core/mcis/manageInfo.go:1040 +0x2aa
github.com/cloud-barista/cb-tumblebug/src/core/mcis.RunRemoteCommand({0xc0013284d3, 0x4}, {0xc000c7c730, 0xa}, {0xc000cd393c, 0x5}, {0x0, 0x0}, {0xc001156120, 0x1, ...})
	/home/son/go/src/github.com/cloud-barista/cb-tumblebug/src/core/mcis/remoteCommand.go:182 +0x193
github.com/cloud-barista/cb-tumblebug/src/core/mcis.RunRemoteCommandAsync(0x0?, {0xc0013284d3, 0x4}, {0xc000c7c730, 0xa}, {0xc000cd393c, 0x5}, {0x0, 0x0}, {0xc001156120, ...}, ...)
	/home/son/go/src/github.com/cloud-barista/cb-tumblebug/src/core/mcis/remoteCommand.go:222 +0x13e
created by github.com/cloud-barista/cb-tumblebug/src/core/mcis.RemoteCommandToMcis
	/home/son/go/src/github.com/cloud-barista/cb-tumblebug/src/core/mcis/remoteCommand.go:160 +0x791
Makefile:8: recipe for target 'run' failed
make: *** [run] Error 2
```

**Environment**
- Source version or branch: main

**Proposed solution**
: NLB 생성시, remoteCommand를 사용하는데, 최근 추가된 bastion node 특징이 영향을 줄 수 있음. 런타임 디버깅 필요.

--- a/src/api/rest/server/mcis/remoteCommand.go
+++ b/src/api/rest/server/mcis/remoteCommand.go
@@ -113,7 +113,7 @@ func RestSetBastionNodes(c echo.Context) error {
 // @Param nsId path string true "Namespace ID" default(ns01)
 // @Param mcisId path string true "MCIS ID" default(mcis01)
 // @Param targetVmId path string true "Target VM ID" default(vm01)
-// @Success 200 {object} mcis.BastionInfo
+// @Success 200 {object} []mcir.BastionNode
 // @Failure 404 {object} common.SimpleMsg
 // @Failure 500 {object} common.SimpleMsg
 // @Router /ns/{nsId}/mcis/{mcisId}/vm/{targetVmId}/bastion [get]

--- a/src/core/mcir/vnet.go
+++ b/src/core/mcir/vnet.go
@@ -135,12 +135,18 @@ func TbSubnetReqStructLevelValidation(sl validator.StructLevel) {
 
 // TbSubnetInfo is a struct that represents TB subnet object.
 type TbSubnetInfo struct { // Tumblebug
-	Id             string
-	Name           string `validate:"required"`
-	IPv4_CIDR      string `validate:"required"`
-	BastionNodeIds []string
-	KeyValueList   []common.KeyValue
-	Description    string
+	Id           string
+	Name         string `validate:"required"`
+	IPv4_CIDR    string `validate:"required"`
+	BastionNodes []BastionNode
+	KeyValueList []common.KeyValue
+	Description  string
+}
+
+// BastionNode is a struct that represents TB BastionNode object.
+type BastionNode struct {
+	McisId string `json:"mcisId"`
+	VmId   string `json:"vmId"`
 }
 
 // CreateVNet accepts vNet creation request, creates and returns an TB vNet object

--- a/src/core/mcis/remoteCommand.go
+++ b/src/core/mcis/remoteCommand.go
@@ -177,10 +177,10 @@ func RunRemoteCommand(nsId string, mcisId string, vmId string, givenUserName str
 		common.CBLog.Error(err)
 		return map[int]string{}, map[int]string{}, err
 	}
-	bastionNode := bastionNodes.VmId[0]
+	bastionNode := bastionNodes[0]
 	// use public IP of the bastion VM
-	bastionIp, _, bastionSshPort := GetVmIp(nsId, mcisId, bastionNode)
-	bastionUserName, bastionSshKey, err := VerifySshUserName(nsId, mcisId, bastionNode, bastionIp, bastionSshPort, givenUserName)
+	bastionIp, _, bastionSshPort := GetVmIp(nsId, bastionNode.McisId, bastionNode.VmId)
+	bastionUserName, bastionSshKey, err := VerifySshUserName(nsId, bastionNode.McisId, bastionNode.VmId, bastionIp, bastionSshPort, givenUserName)
 	bastionEndpoint := fmt.Sprintf("%s:%s", bastionIp, bastionSshPort)
 
 	bastionSshInfo := sshInfo{
@@ -536,7 +536,7 @@ func SetBastionNodes(nsId string, mcisId string, targetVmId string, bastionVmId 
 		common.CBLog.Error(err)
 		return "", err
 	}
-	if len(currentBastion.VmId) > 0 && bastionVmId == "" {
+	if len(currentBastion) > 0 && bastionVmId == "" {
 		return "", fmt.Errorf("bastion node already exists for VM (ID: %s) in MCIS (ID: %s) under namespace (ID: %s)",
 			targetVmId, mcisId, nsId)
 	}
@@ -576,16 +576,17 @@ func SetBastionNodes(nsId string, mcisId string, targetVmId string, bastionVmId 
 					}
 				}
 			} else {
-				for _, existingId := range subnetInfo.BastionNodeIds {
-					if existingId == bastionVmId {
+				for _, existingId := range subnetInfo.BastionNodes {
+					if existingId.VmId == bastionVmId {
 						return fmt.Sprintf("Bastion (ID: %s) already exists in subnet (ID: %s) in VNet (ID: %s).",
 							bastionVmId, subnetInfo.Id, vmObj.VNetId), nil
 					}
 				}
 			}
 
+			bastionCandidate := mcir.BastionNode{McisId: mcisId, VmId: bastionVmId}
 			// Append bastionVmId only if it doesn't already exist.
-			subnetInfo.BastionNodeIds = append(subnetInfo.BastionNodeIds, bastionVmId)
+			subnetInfo.BastionNodes = append(subnetInfo.BastionNodes, bastionCandidate)
 			tempVNetInfo.SubnetInfoList[i] = subnetInfo
 			mcir.UpdateResourceObject(nsId, common.StrVNet, tempVNetInfo)
 
@@ -598,8 +599,8 @@ func SetBastionNodes(nsId string, mcisId string, targetVmId string, bastionVmId 
 }
 
 // GetBastionNodes func retrieves bastion nodes for a given VM
-func GetBastionNodes(nsId string, mcisId string, targetVmId string) (BastionInfo, error) {
-	returnValue := BastionInfo{}
+func GetBastionNodes(nsId string, mcisId string, targetVmId string) ([]mcir.BastionNode, error) {
+	returnValue := []mcir.BastionNode{}
 	// Fetch VM object based on nsId, mcisId, and targetVmId
 	vmObj, err := GetVmObject(nsId, mcisId, targetVmId)
 	if err != nil {
@@ -624,10 +625,10 @@ func GetBastionNodes(nsId string, mcisId string, targetVmId string) (BastionInfo
 	// Find the subnet corresponding to the VM and return the BastionNodeIds
 	for _, subnetInfo := range tempVNetInfo.SubnetInfoList {
 		if subnetInfo.Id == vmObj.SubnetId {
-			if subnetInfo.BastionNodeIds == nil {
+			if subnetInfo.BastionNodes == nil {
 				return returnValue, nil
 			}
-			returnValue.VmId = subnetInfo.BastionNodeIds
+			returnValue = subnetInfo.BastionNodes
 			return returnValue, nil
 		}
 	}

